Update ranged enemy sections

ntwbruce opened the pull request:
Update the following scenes: Ranged Dragon Enemy Introduction, Dragon Riding Tutorial, Knight Enemy Ranged Introduction, Ranged Enemies Combined. Modifies the prefabs for these scenes, as well as the overall level.

- Add obstacles / platforms.
- Dragon Riding Tutorial now includes breakable walls and enemies. 
- Ranged Enemies Combined has more enemies.

ntwbruce committed:
Update dragon riding tutorial

- Add enemies + gate, 2 sections where the knight has to mount/dismount

ntwbruce committed:
Merge branch 'master' into branch-levelexpand

ntwbruce committed:
Update level section manager

Forgot to update the Photon View so the enemy spawning messed up.

ntwbruce committed:
Add extra elements to ranged enemy introduction sections

ntwbruce committed:
Update positions for obstacles in combined ranged enemy section

ntwbruce committed:
Update positions of obstacles

ntwbruce committed:
Update positioning in combined ranged enemy section again

ntwbruce committed:
Modify knight enemy spawn position in riding tutorial section

ntwbruce committed:
Update positions of sections in level scene

Also update pathfinding grid and camera and physical boundaries.

kenneth-fung reviewed (changes requested):
Looks good, well done. I'd like to make three minor requests.

Can the ranged dragon enemy (in its introduction section) spawn point be moved onto the platform in the middle? Otherwise the dragon needs to move some distance into the level before it starts aiming.

![image](https://user-images.githubusercontent.com/50415999/162972548-3ee5baf5-866d-4d68-bd80-8a302d2c2337.png)

There's a small gap at the top of the lower breakable wall in the dragon riding section.

![image](https://user-images.githubusercontent.com/50415999/162972864-e1ae62d1-e18b-43af-9903-1ee652f34160.png)

For the platforms in all the sections edited, can you move them downwards just slightly so that the tops aren't right up against the grid's lines? This helps some of the grounded AI actors a bit when jumping up these platforms. You can refer to the platforms in the other sections, it's just a slight gap so the platform is more in the center of the grid row.

ntwbruce committed:
Fix positioning

- Position of ranged dragon enemy in introduction section
- Position of lower breakable wall in riding tutorial section
- Position of platforms

ntwbruce committed:
Change positioning of knight enemy in riding tutorial

Original position allows for cheesing: the knight can ride the dragon
and drop behind the enemy, and it cannot see the knight.

kenneth-fung reviewed (approved):
Looks good, thanks for changes.
